setup.cfg

```diff
@@ -37,7 +37,7 @@ long_description_content_type = text/markdown
     requests
     beautifulsoup4
     lxml
-    pysatCDF
+    cdflib == 0.3.19
```

Comment:
See top comment

Reply:
Oh, also, setup input isn't supposed to have version limitations.

Reply:
I remember you mentioning this. Should have changed it sooner. Does putting the version here do anything?

Reply:
I think it does, but here is where I learned about not putting those in the config: https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
Ah! I completely missed the part about not pinning it to a specific version. I think I've updated it to be consistent now. I went ahead and tested 0.3.18 and 0.3.17 and they both seem to work so it was silly to pin it to a specific version anyway.